Atom (RSS) Feed for Videos at /feed.xml

pootsbook opened the pull request:
* XML taken from Jekyll generated template
* Ampersand HTML entity used for XML conformance
* site.url configuration added for absolute link
* Images not added due to 1:1, 2:1 ratio requirements
* Talks flattened for simplicity

karlfreeman commented:
This looks really great! :+1:

Would be lovely if you could add a `link` tag to [`head.html`](https://github.com/emberlondon/emberlondon.github.io/blob/master/_includes/head.html) referencing `/feed.xml`.

jgwhite commented:
This is amazing. Add the link to `<head>` and this is good to go.

pootsbook commented:
Added the Atom link to head.

Rebased against latest `master`, I hope I've done the right thing. If not, then cherry-picking 585927b and 297b874 should do the trick.
